manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -187,12 +210,41 @@ public function getExtensionConfig($extensionName, array $extensionConfigs, Plug
     }
 
     /**
-     * Sets path to enable autoloading of legacy Contao modules.
+     * Sets the PDO driver options, if applicable (#2459).
      *
-     * @param string $modulePath
+     * @return array
      */
-    public static function autoloadModules($modulePath)
+    private function setPdoDriverOptions(array $extensionConfigs, PluginContainerBuilder $container)
     {
-        static::$autoloadModules = $modulePath;
+        // Do not add PDO options, if constant does not exist
+        if (!\defined('PDO::MYSQL_ATTR_MULTI_STATEMENTS')) {
+            return $extensionConfigs;
+        }
+
+        foreach ($extensionConfigs as $extensionConfig) {
+            // Do not add PDO options, if selected driver is not pdo_mysql
+            if (isset($extensionConfig['dbal']['connections']['default']['driver']) && 'pdo_mysql' !== $extensionConfig['dbal']['connections']['default']['driver']) {
+                return $extensionConfigs;
+            }
```

Comment:
A subsequent extension config could set the driver back to `pdo_mysql`. Should we consider this possibility? Seems unlikely though.

Reply:
Our only real problem is that the constants don't exist, but we check for that. So I'd be fine with that.